project/plugins.sbt

```diff
@@ -1,8 +1,10 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")
-addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.0")
+addSbtPlugin("com.github.sbt" % "sbt-ghpages" % "0.8.0")
+addSbtPlugin(
+  "com.github.sbt" % "sbt-site" % "1.5.0" // TODO: sbt-site-sphinx, see https://github.com/sbt/sbt-site/issues/186
+)
```

Comment:
@muuki88 Unfortunately, sbt-site dropped the sphinx module, at least temporary (which is needed for [the docs](https://github.com/sbt/sbt-native-packager/tree/master/src/sphinx) I guess), at least until someone steps up to revive it:
- https://github.com/sbt/sbt-site/issues/186

What do you think? Should someone invest time to fix it (not sure if that is hard, maybe it's an easy fix) or should sbt-native-packager move to some other documentation format?

Reply:
Fixed by https://github.com/sbt/sbt-site/pull/214